scripts/bash/lxd-client

```diff
@@ -96,6 +96,7 @@ _have lxc && {
       security.syscalls.blacklist_compat security.syscalls.blacklist_default \
       security.syscalls.intercept.mknod security.syscalls.intercept.mount \
       security.syscalls.intercept.mount.allowed \
+      security.syscall.intercept.mount.fuse \
       security.syscalls.intercept.setxattr \
       security.syscall.intercept.mount.shift \
```

Comment:
Typo on both mount.fuse and mount.shift, it's `syscalls` not `syscall`.

Reply:
fixed and rebased